docs/news.rst

```diff
@@ -5,21 +5,28 @@ Release notes
 
 1.3.0
 -----
-*Unreleased*
 
 Added
 ~~~~~
 
 - Jobs website shortcut to cancel a job using the cancel.json webservice.
 - Make project argument to listjobs.json optional,
   so that we can easily query for all jobs.
-- Python 3.7, 3.8 and 3.9 support
+- Python 3.7, 3.8 and 3.9, 3.10 support
+- scrapyd can now run with asyncioreactor, changes in runner
+
 - Configuration option for job storage class
 - Configuration option for egg storage class
+- support for HTTP authentication in scrapyd server
+- replaced deprecated SafeConfigParser with ConfigParser
+- much improved test coverage
+- improved HTTP headers in webservice
 
 Removed
 ~~~~~~~
 
+- Python 2 support
+- Python 
```

Comment:
Missing content?

Reply:
yes sorry, christmas mood will fix

Reply:
I updated news, now just waiting for pypi token and we can release @Gallaecio can you add secret pypi token here?

Reply:
`PYPI_TOKEN` defined.